src/components/partials/tables/FTable.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+    <div class="q-px-none q-py-sm table-pagination">
+        <div
+            v-if="pageHeader"
+            class="row"
+        >
+            <div class="flex col-xs-12 col-md-6 q-pa-md">
+                <slot name="header-left">
+                    <span
+                        v-if="headerHelpText"
+                        class="text-grey-8"
+                        v-text="headerHelpText"/>
+
+                    <q-breadcrumbs
+                        v-else-if="breadcrumbs.length > 0">
+                        <template v-slot:separator>
+                            <q-icon
+                                size="1.3em"
+                                name="chevron_right"
+                                color="grey-7"
+                            />
+                        </template>
+                        <q-breadcrumbs-el
+                            v-for="breadcrumb in breadcrumbs"
+                            :label="$trans(breadcrumb.label)"
+                            :key="breadcrumb.label"
+                            :to="breadcrumb.to"
+                            class="text-grey-7"
+                        />
+                    </q-breadcrumbs>
+
+                </slot>
+            </div>
+            <div class="col-xs-12 col-md-6">
+                <div class="float-right">
+                    <slot name="header-right" />
+                </div>
+            </div>
+        </div>
+
+        <div class="row lt-md">
+            <div class="col-xs-12 border-bottom-grey">
+                <div class="float-right q-mb-sm q-mx-sm">
+                    <slot name="control-right" />
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div
+                :class="tabIndicatorColor === 'transparent' ? 'q-pa-sm' : ''"
+                class="col-xs-12 col-md-6 border-bottom-grey "
+            >
+                <slot name="control-left">
+
+                    <div
+                        v-if="tableTitle"
+                        class="text-h5 q-pa-md"
+                        v-text="tableTitle"
+                    />
+                    <q-tabs
+                        v-else
+                        v-model="selectedTab"
+                        :indicator-color="tabIndicatorColor"
+                        :active-color="tabIndicatorColor === 'transparent' ? 'white' : 'primary'"
+                        :class="tabIndicatorColor === 'transparent' ? 'pa-xs' : ''"
```

Comment:
Including the L51, since we often use the `tabIndicatorColor === 'transparent'`, I think we can create a function for this.
Example:
```js
sampleFunction(data1, defaultData = '') {
  return tabIndicatorColor === 'transparent'
    ? data1
    : defaultData;
}
```

Then the usage is:
```js
  :active-color="sampleFunction('white', 'primary')"
 
  //FOR CLASS USAGE
  :class="sampleFunction('pa-xs')"
```

Reply:
This should also be achieved by using more specific CSS, a function is unnecessary

Reply:
@pandafox yes, add CSS for that.

Reply:
As discussed in [slack](https://freedom.slack.com/archives/G5GFA0P6F/p1618392818038100), custom styling and behavior will be handled by the parent component for now. Theming support will be implemented in another ticket.

Reply:
yes. Thanks.